docs/content/manual/pre-release/node/node-disconnection.md

```diff
@@ -4,14 +4,58 @@ title: Node disconnection test
 https://github.com/longhorn/longhorn/issues/1545
 For disconnect node : https://github.com/longhorn/longhorn/files/4864127/network_down.sh.zip
 
+If auto-salvage is disabled, the auto-reattachment behavior after the node disconnection depends on all replicas are in ERROR state or not.
+
+(1) If all replicas are in ERROR state, the volume would remain in detached/faulted state if auto-salvage is disabled.
+
+(2) If there is any healthy replica, the volume would be auto-reattached even though auto-salvage is disabled.
+
+What makes all replicas in ERROR state? If there is data writing during the disconnection, due to the engine process not able to talk with other replicas, the engine process will mark all other replicas as ERROR.
+
+So we have 2 test cases for node disconnection + disabled auto-salvage:
+
+(Case 1) data writing during the disconnection -> all replicas become ERROR -> no auto-reattach after the disconnection
+
+(Case 2) no data writing during the disconnection -> other replicas are still healthy -> auto-reattach after the disconnection
+
 ### Case 1:
-1. Disable the setting auto-salvage.
-2. Create and attach a volume.
-3. Keep writing data to the volume. Disconnect the node that the volume attached to for 100 seconds during the data writing.
-4. Wait for the node back.
-5. The volume will be detached then reattached automatically. And there are some replicas still running after the reattachment.
+1. Disable auto-salvage.
+2. Create a volume named test-1.
+3. Attach the volume to a node.
+4. Keep writing data directly to the volume. (sudo dd if=/dev/urandom of=/dev/longhorn/test-1 bs=2M count=2048)
+5. Disconnect the network of the node that the volume attached to for 100 seconds during the data writing. (sudo nohup ./network_down.sh 100)
+6. Wait for the node back.
+7. The volume should remain detached, and all replicas remain in ERROR state.
 
 ### Case 2:
+1. Disable auto-salvage.
+2. Create a volume named test-1.
+3. Create PV/PVC for the volume through longhorn UI.
+4. Create a pod with the volume. (kubectl apply -f pod.yaml)
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: test-pod
+spec:
+  containers:
+    - name: sleep
+      image: busybox
+      imagePullPolicy: IfNotPresent
+      args: ["/bin/sh", "-c", "while true;do date;sleep 5; done"]
+      volumeMounts:
+        - name: pod-data
+          mountPath: /data
+  volumes:
+    - name: pod-data
+      persistentVolumeClaim:
+        claimName: test-1
+```
+5. Disconnect the network of the node that the volume attached to for 100 seconds during the data writing (sudo nohup ./network_down.sh 100)
```

Comment:
I thought what you want to describe is, right?

Disconnect the network of the node that the volume attached to for 100 seconds and during the time, there is no any data writing (sudo nohup ./network_down.sh 100)